### PR TITLE
Remove event type from cloudhub

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:
Now we use `beehiveModel.Message` to send message between cloudhub and edgehub, this PR removed the `event type` from cloudhub. There's no need to change message to event and event to message again.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cloud/pkg/cloudhub/common/model/types.go [hunk 1]
```diff
@@ -61,48 +61,6 @@ type HubInfo struct {
...
-// Event represents message communicated between cloud hub and edge hub
-type Event struct {
-	Group     string        `json:"msg_group"`
-	Source    string        `json:"source"`
-	UserGroup UserGroupInfo `json:"user_group"`
-	ID        string        `json:"msg_id"`
-	ParentID  string        `json:"parent_msg_id"`
-	Timestamp int64         `json:"timestamp"`
-	Content   interface{}   `json:"content"`
-}
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/common/model/types.go]: Remove the `Event` struct.

File: cloud/pkg/cloudhub/cloudhub.go [hunk 2]
```diff
@@ -41,18 +41,18 @@ func (a *cloudHub) Start(c *beehiveContext.Context) {
 
 	initHubConfig()
 
-	eventq := channelq.NewChannelEventQueue(c)
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/cloudhub.go]: `messageq` or `msgQueue`?

File: cloud/pkg/cloudhub/cloudhub.go [hunk 3]
```diff
@@ -41,18 +41,18 @@ func (a *cloudHub) Start(c *beehiveContext.Context) {
...
+	messageq := channelq.NewChannelMessageQueue(c)
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/cloudhub.go]: `messageq` or `msgQueue`?

**Comment by kadisi** [comment]
@fisherxu  please rebase your pr

**Comment by fisherxu** [comment]
> @fisherxu please rebase your pr

Have done.

File: cloud/pkg/cloudhub/handler/messagehandler.go [hunk 4]
```diff
@@ -40,52 +40,52 @@ const (
...
-		CloudhubHandler.Handlers = []HandleFunc{CloudhubHandler.KeepaliveCheckLoop, CloudhubHandler.EventWriteLoop}
+		CloudhubHandler.Handlers = []HandleFunc{CloudhubHandler.KeepaliveCheckLoop, CloudhubHandler.MessageWriteLoop}
 
 		CloudhubHandler.initServerEntries()
 	})
 }
 
 // initServerEntries register handler func
-func (eh *EventHandle) initServerEntries() {
+func (eh *MessageHandle) initServerEntries() {
```
Comment (Baoqiang-Zhang) [inline cloud/pkg/cloudhub/handler/messagehandler.go]: eh should be change to mh

Comment (fisherxu) [reply cloud/pkg/cloudhub/handler/messagehandler.go]: Done.

**Comment by sids-b** [comment]
Thanks @fisherxu 

/lgtm

**Pull request merged**